### PR TITLE
{Compute} Doc fix for 'vm user delete'

### PR DESCRIPTION
**Pull request by paulmey**
Clarify that 'vm user delete' also removes the home directory on Linux systems.

**History Notes:**  
[Compute] (az vm user delete:) Clarify that 'vm user delete' also removes the home directory on Linux systems.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by jasonzio** [comment]
Matching change to the [web page documentation](https://docs.microsoft.com/en-us/cli/azure/vm/user) is also required.

File: src/azure-cli/azure/cli/command_modules/vm/_help.py [hunk 1]
```diff
@@ -1950,6 +1950,8 @@
 helps['vm user delete'] = """
 type: command
 short-summary: Delete a user account from a VM.
+long-summary: >
+    Also deletes the user home directory on Linux VMs.
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/vm/_help.py]:
```suggestion
    Also delete the user home directory on Linux VMs.
```

**Pull request merged**